projects/igniteui-angular/src/lib/core/styles/components/calendar/_calendar-theme.scss

```diff
@@ -594,6 +716,31 @@
                 padding-bottom: $cal-picker-padding;
                 padding-top: $cal-picker-padding;
             }
+
+            @if $not-bootstrap-theme{
+                margin-bottom: 0;
+            }
+        }
+
+        &:nth-child(2) {
+            @if $not-bootstrap-theme{
+                margin-top: 0;
+            }
+        }
+
+        &:last-of-type {
+            %cal-value-content--week-number {
+                border-bottom-left-radius: $border-radius;
+                border-bottom-right-radius: $border-radius;
```

Comment:
RTL.

Reply:
This is not affected by RTL. The radius is the same for all corners.